FreshRSS/FreshRSS
# Improved: Reorg slider buttons

Ref. #3380 (number 2)

Changes proposed in this pull request:

- added a trash icon
- moved the `remove` button upper
- the filter and statistic link looks now like a button
- this 3 buttons are now in the same row
- changed the icon for the `filter` link/button: from `mouse cursor `icon to the `eye `icon

Before:
![grafik](https://github.com/user-attachments/assets/a76061dc-1c6e-4bee-98b2-7be16eac7c70)

After:
![grafik](https://github.com/user-attachments/assets/c5b42611-f22d-4518-ace4-3f600fcc7ae2)


How to test the feature manually:

1. open the feed config
2. check the filter button
3. check the statistic button
4. check the remove/delete button

Pull request checklist:


Todos left:
- [ ] style the trash icon button in the other themes too (`.btn-attention`)
- [ ] config category
- [ ] config user query
- [ ] config label management
